apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -231,9 +239,11 @@ export default function ListBox({
     // If values have been filtered in the currently loaded page, we want to
     // prevent rendering empty rows by assigning the actual number of items to render
     // since count (qcy) does not reflect this in DQ mode currently.
-    const hasFilteredValues = ps.some((page) => page.qArea.qHeight < MINIMUM_BATCH_SIZE);
+    // If any qMatrix was not 100 length && result of it was less that qcy
+    // then => qTop + qHeight might indicate length of items as long as you scroll and fetch more items
+    const notEnoughItems = ps.some((p) => p.qArea.qHeight !== MINIMUM_BATCH_SIZE);
```

Comment:
Not sure if renaming this makes it easier to understand

Reply:
if the batch size is not 100, then we might know that we are at the end of list => `notEnoughItems` 
at the end we check for the amount of items (all batches so far) with original amount (qcy) => this will gives us the insight if we are selected some thing or not 

because if we do, then the amount should be less that the original

Reply:
✅ ok, changed it to `hasFilteredValues`